spec/javascript/components/supplemental_application/sections/PreferencesTable.test.js

```diff
@@ -23,7 +22,6 @@ describe('PreferencesTable', () => {
       />
     )
 
-    let tree = component.toJSON()
-    expect(tree).toMatchSnapshot()
+    expect(wrapper.find('#alice-griffith-housing-development-resident-row').exists()).toBeTruthy()
```

Comment:
@chrisdolendo are there any rows that shouldn't be there that we can check? I'm not clear on what all of the expected prefs are or are not.